vllm/worker/cpu_model_runner.py

```diff
@@ -361,3 +387,46 @@ def execute_model(
             sampling_metadata=sampling_metadata,
         )
         return output
+
+    @torch.inference_mode()
+    def profile_run(self) -> None:
+        # Warming up the model with batchsize = [1, max_num_seqs,
+        # max_num_batched_tokens], to generate corresponding operators
+        # using torch.compile.
+
+        model_config = self.model_config
+        vlm_config = self.vision_language_config
+
+        # Enable top-k sampling to reflect the accurate memory usage.
```

Comment:
nit: This comment is a bit confusing since we are not profiling memory here.
```suggestion
```

Reply:
Removed